app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -837,11 +837,11 @@ class Settings(private val appContext: Context) : PreferencesHolder {
             camera = getSitePermissionsPhoneFeatureAction(PhoneFeature.CAMERA),
             autoplayAudible = getSitePermissionsPhoneFeatureAutoplayAction(
                 feature = PhoneFeature.AUTOPLAY_AUDIBLE,
-                default = AutoplayAction.ALLOWED
+                default = AutoplayAction.BLOCKED
             ),
             autoplayInaudible = getSitePermissionsPhoneFeatureAutoplayAction(
                 feature = PhoneFeature.AUTOPLAY_INAUDIBLE,
-                default = AutoplayAction.BLOCKED
+                default = AutoplayAction.ALLOWED
```

Comment:
@Mugurell this PR corrects the default values of [my previous PR ](https://github.com/mozilla-mobile/fenix/pull/20073/files#diff-0e4df8f67ec9bfb7f74da9d75376085d7ecd5a5737322bf8375e387618bec5f3R838-R845), where by mistake I was allowing `autoplayAudible` and blocking `autoplayInaudible`. When I should be blocking `autoplayAudible` and allowing `autoplayInaudible`. As the main goal it's to "block audio only" by default.

Reply:
For this reason, I had to change the key to reset the values.